docs/core_and_modules.rst

```diff
@@ -1151,7 +1151,15 @@ List of metrics group names and particular metrics in them:
     * measure - default: call - metric value is a command or script execution output. Example: `<Custom measure="call" diff="1" label="Base size">du -s /var/lib/mysql/ | awk '{print $1}'</Custom>`
 * TelegrafRaw
     * raw telegraf TOML format, transparently added to final collector config 
-* Source additional source file in telegraf json format, can be used to add custom metrics that needs complex processing and do not fit into standart custom metrics (like log parsing with aggregation)
+* Source
+    additional source file in telegraf json format,     can be used to add custom metrics that needs complex processing and do not fit into standart custom metrics (like log parsing with aggregation)
```

Comment:
custom metrics do not include timestamps and this source does. This is the main difference. You can not import asynchronous data with custom metrics, but you can with this Source. Please, mention that.

Reply:
Also, did you forgot the '*' before the second line?